amd/src/crossword_question.js

```diff
@@ -446,4 +447,27 @@ export class CrosswordQuestion {
         // Set wordNumber by default value.
         this.options.wordNumber = -1;
     }
+
+    /**
+     * Calculate cell size based on the number of columns and rows,
+     * If rows or columns more than 15, we will set size 40px (default).
+     *
+     * @param {Array} option The crossword option.
+     * @return {Array} The crossword option after change the cell size.
+     */
+    calculateCellSize(option) {
+        let size = option.cellSize[0];
+        option.isSmall = 0;
+        if (option.colsNum > 15 || option.rowsNum > 15 || window.innerWidth < 768) {
+            size = option.cellSize[1];
+            option.isSmall = 1;
+        }
+        // Calculate the size of the crossword: we need to add space between the cells.
+        // The size of the crossword is calculated by the formula:
+        // number of cells multiplied by (1 cell length + 1) and then 1.
```

Comment:
No need to duplicate the code in the comment, so don't repeat the formula. The first line of the comment is what is useful. (You could make it clearer by changing it to something more precise like "The crossword size is the size of the cells plus the 1px borders."

What is not clear from the code, until you think about it, is that the +1 is becuase there is a 1px border between the cells. That is what the comment should say.